frequently-asked-questions.md

```diff
@@ -121,10 +129,17 @@ CockroachDB is not yet suitable for real-time analytics, although support for an
 
```

Comment:
You should also mention use cases which require joins in SQL (but qualify that by saying they're slated for the next major release). I would also say that CockroachDB is not a good choice when very low latency reads and writes are critical; use an in-memory database instead.